src/Indice.Features.Cases.AspNetCore/Services/CaseMessageService/BaseCaseMessageService.cs

```diff
@@ -74,7 +74,7 @@ internal abstract class BaseCaseMessageService
                 if (!message.PrivateComment.HasValue) {
                     message.PrivateComment = newCheckpointType.Private;
                 }
-                var suffix = $"Case status changed to '{newCheckpointType.Name}'";
+                var suffix = $"Case status changed to '{newCheckpointType.Code}'";
```

Comment:
Αυτό παλιίο λάθος αλλά το `Case status...` νομίζω πρέπει να γίνει `Setting case to {code}` ή κάτι που να αναφέρεται στα checkpoints anyway